src/main/resources/config.yml

```diff
@@ -97,48 +97,55 @@ backup-modes:
     dir: '/'
     allow-manual: true
     schedule: 0
+    purge-after: 0
```

Comment:
Would be good to add a note to the above comment, saying

- The time unit of the `purge-after` time interval is _minutes_
- Set to `0` or negative to disable (see my other comments)

Reply:
Done!